docs/realtime/Protocol.md

```diff
@@ -0,0 +1,323 @@
+# RTC protocol
+
+The WebSocket API provides means to receive events on change resources (that the JWT has access to).
+
+## Connecting to the WebSocket API
+
+Build the URL according to the template below:
+
+```
+wss://<project-id>.<zone>.app.jexia.com/rtc?access_token=<jwt>
+```
+ARGUMENT     | TYPE   | DESCRIPTION                           | EXAMPLE
+-------------|--------|---------------------------------------|----------------------------------------
+project-id   | UUID   | unique identifier/hash of the project | `4f8c6ee3-f2d1-47b8-83c4-c5250b86cab4`
+zone         | string | project zone                          | `nl00`
+jwt          | string | valid token obtained after logging into the project using the API key credentials or any other method | `eyJhbGciOiJS... ...aev96IlwB1OFzDmyw`
+
+Connecting to the WebSocket API is done using a WebSocket client and connect it to the WebSocket address. Use one of the available WebSocket clients to create a connection:
+- [Simple WebSocket Client](https://chrome.google.com/webstore/detail/simple-websocket-client/pfdhoblngboilpfeibdedpjgfnlcodoo) (Chrome extension)
+- [Smart Websocket Client](https://chrome.google.com/webstore/detail/smart-websocket-client/omalebghpgejjiaoknljcfmglgbpocdp) (Chrome extension)
+- [WebSocket King](https://websocketking.com) (Online WebSocket client)
+
+## Before you start
+1. Create a User/API key.
+2. Log In providing valid credentials and method (check [here](../auth/README.md) for documentaion).
```

Comment:
> ../auth/README.md

Is this the correct URL to link to another (web)page? (After the Markdown is rendered into HTML)

Reply:
do you mean we cannot use a relative path?
(relative path is valid if you click on the link it follows to https://github.com/jexia/docs/blob/issue/JEXIA-3985-rtc-docs/docs/auth/README.md)